deploy/docs/Non_Helm_Installation.md

```diff
@@ -85,19 +85,29 @@ The following parameter is optional, but we recommend setting it.
 - **sumologic.clusterName** - An identifier for your Kubernetes cluster.
   This is the name you will see for the cluster in Sumo Logic. Default is `kubernetes`.
 
-First you will generate the YAML to apply to your cluster.  The following command contains the minimum parameters that can generate the YAML to setup Sumo Logic's Kubernetes collection. This command will generate the YAML and pipe it a file called `sumologic.yaml`. Please note that `--namespace` is required
+Create `values.yaml` with the configuration. For example, the minimal one will look like the following:
+
+```yaml
+sumologic:
+  accessId: ${ACCESS_ID}
+  accessKey: ${ACCESS_KEY}
+  clusterName: ${CLUSTER_NAME}
+```
+
+Now you will need to generate the YAML to apply to your cluster.
+The following command contains the minimum parameters that can generate the YAML to setup Sumo Logic's Kubernetes collection.
```

Comment:
Is this line necessary? We already said it was the minimal configuration above.

Reply:
I understand by minimal parameters that we refers to `helm` command. Imho it's not necessary to mention that. I can remove it

Reply:
I think it's kind of confusing then, and we don't lose anything by removing it.